ci/scripts/misc-check-cron.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Exits as soon as any line fails.
+set -euo pipefail
+
+source ci/scripts/common.sh
+
+# RUSTSEC-2023-0052: https://github.com/risingwavelabs/risingwave/issues/11842
+# RUSTSEC-2023-0071 https://github.com/risingwavelabs/risingwave/issues/13703
+echo "--- Run audit check"
+cargo audit \
+  --ignore RUSTSEC-2023-0052 \
+  --ignore RUSTSEC-2023-0071
```

Comment:
Could we use a config file for it?

https://github.com/risingwavelabs/risingwave/pull/13716/files